File: backend/sti-protection/src/main/java/de/eshg/stiprotection/pdf/identification/Appointment.java
/*
 * Copyright 2024 cronn GmbH
 * SPDX-License-Identifier: AGPL-3.0-only
 */

package de.eshg.stiprotection.pdf.identification;

public record Appointment(
    Department department,
    String date,
    String time,
    String durationMinutes,
    String room,
    String url,
    String ticketNumber) {}

File: employee-portal/src/lib/businessModules/schoolEntry/api/models/Appointment.ts
/**
 * Copyright 2025 cronn GmbH
 * SPDX-License-Identifier: AGPL-3.0-only
 */

import { ApiAppointment } from "@eshg/employee-portal-api/schoolEntry";

export interface Appointment {
  readonly start: Date;
  readonly end: Date;
}

export function mapAppointment(response: ApiAppointment): ApiAppointment {
  return { start: response.start, end: response.end };
}
